src/StaticFAQ/FAQExtraInfo/BaggageInfo/index.js

```diff
@@ -81,13 +81,13 @@ class BaggageInfo extends React.Component<Props> {
     return (
       <div className="baggageCard">
         <div className="iconTitle">
-          <Baggages customColor="black" />
+          <BaggageChecked customColor="black" />
         </div>
         <h1 className="title">Your baggage</h1>
         <div className="subtitle">
           <Text type="attention">Here you can see your baggage allowance.</Text>
         </div>
-        <BaggageSummary data={baggage} mmbUrl={directAccessURL} />
+        <BaggageSummary data={baggage || null} mmbUrl={directAccessURL} />
```

Comment:
if no baggage is allowed - a list is empty, nothing is rendered if I'm correct. I would add some note with italic (as it's here https://zpl.io/aN09rwe for "You cannot add more baggage to this booking.") saying you don't have any baggage allowance

Reply:
+ would be nice to add to some unit tests for this, covering at least these two cases - 1) you have baggage 2) you don't have any baggage

Reply:
> if no baggage is allowed - a list is empty, nothing is rendered if I'm correct. I would add some note with italic (as it's here https://zpl.io/aN09rwe for "You cannot add more baggage to this booking.") saying you don't have any baggage allowance

Good idea, however, is having zero baggage allowance possible at all?

Reply:
> would be nice to add to some unit tests for this, covering at least these two cases - 1) you have baggage 2) you don't have any baggage

Good idea!

Reply:
:heavy_check_mark: tests added